djpsa/halo/records/budgetdata/model.py

```diff
@@ -0,0 +1,50 @@
+from django.db import models
+from model_utils import FieldTracker
+
+# This is not a real record in Halo, but created for the purpose
+# of easily storing and processing budget data. Uses the Ticket
+# API to get a tickets budget data instead of having its own.
+
+
+class BudgetData(models.Model):
+    ticket = models.ForeignKey(
+        'Ticket', on_delete=models.CASCADE, blank=True, null=True)
```

Comment:
Is null=True appropriate here? What use is a budget with no ticket?

Reply:
I've just been blanket making everything null=True for Halo because I don't trust their API at all, I think I mentioned that at one point. BUT, this is a fair case for not being a problem here.